website/other/bezier-rs-demos/wasm/src/bezier.rs

```diff
@@ -15,11 +15,6 @@ struct CircleSector {
 }
 
 #[derive(Serialize, Deserialize)]
```

Comment:
Forgot to delete this line, unless you intend for it to be added to the enum below.

Reply:
Great catch, good eye!